ext/crypto/01_webidl.js

```diff
@@ -1,516 +0,0 @@
-// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
```

Comment:
There was a circular dependency on `CryptoKey` between this file and `00_crypto.js` (which uses relevant WebIDL converters). I moved this file into `00_crypto.js`.